dss/config.py

```diff
@@ -216,7 +216,7 @@ def get_es_index_name(index_type: ESIndexType, replica: Replica,
     def get_es_alias_name(index_type: ESIndexType, replica: Replica) -> str:
         """Returns the alias for indexes"""
         deployment_stage = os.environ["DSS_DEPLOYMENT_STAGE"]
-        index = f"dss-{index_type.name}-{deployment_stage}-{replica.name}"
+        index = f"dss-{index_type.name}-{replica.name}-{deployment_stage}-alias"
```

Comment:
dss-{stage}-{replica}-{type}-{schema} for indices
dss-{stage}-{replica}-{type}-alias for the alias

Reply:
updated